### PR TITLE
Adds Waitematā, correctly spelled.

### PR DESCRIPTION
**Pull request by andycarrell**
@finnclark this also sets up a map from dhb name to whatever we want - I've only changed Waitematā so far but keen for more suggestions

### CODE REVIEW

File: app/utilities/dhbDisplayName.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  Tairawhiti: "Tairawhiti",
+  Taranaki: "Taranaki",
+  Waikato: "Waikato",
+  Wairarapa: "Wairarapa",
+  Waitemata: "Waitematā",
+  "West Coast": "West Coast",
+};
+
+const dhbDisplayName = (raw: string): string => {
+  console.log(raw, nameMap[raw], raw in nameMap);
```
Comment (theramis) [inline app/utilities/dhbDisplayName.ts:29]: Did you want to leave this in?

File: app/utilities/dhbDisplayName.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "Bay of Plenty": "Bay of Plenty",
+  Canterbury: "Canterbury",
+  "Capital and Coast": "Capital and Coast",
+  "Counties Manukau": "Counties Manukau",
+  "Hawkes Bay": "Hawkes Bay",
+  "Hutt Valley": "Hutt Valley",
+  Lakes: "Lakes",
+  MidCentral: "MidCentral",
+  "Nelson Malborough": "Nelson Malborough",
+  "Overseas / Unkown": "Overseas / Unkown",
```
Comment (theramis) [inline app/utilities/dhbDisplayName.ts:16]:
```suggestion
  "Overseas / Unknown": "Overseas / Unknown",
```

**Review by theramis — APPROVED** [review]
Smol changes, merge whenever you are happy.

File: app/utilities/dhbDisplayName.ts (hunk 1 above)
Comment (andycarrell) [reply app/utilities/dhbDisplayName.ts:29]: NO!

console.log considered harmful

**Review by finnclark — APPROVED** [review]
Heck yeah!

File: app/utilities/dhbDisplayName.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+type NameMapType = {
+  [key: string]: string;
+};
+
+const nameMap: NameMapType = {
```
Comment (jishaal) [inline app/utilities/dhbDisplayName.ts:5]: Can do `const nameMap: Record<string, string>`

Ideally once all this is merged we create a type so we are consistent with the serverside name, so would be something like 

`const nameMap: Record<DhbName, string>`

Comment (andycarrell) [reply app/utilities/dhbDisplayName.ts:5]: Liiiiit exactly what I'm looking for

**Review by jishaal — APPROVED** [review]

**Pull request merged**